### DISCUSSION TITLE
How long will it take to give me the benefits of Github Education?

### DISCUSSION THREAD

**Discussion by carloss-vk** [community/community#58858] [GitHub Education]
### Hi everyone,

How long will it take to give me the benefits of Github Education?

**Comment by dakshdeepHERE** [comment]
You instantly get all the benefits once your verification is completed. If you want the additional benefits then check this [link](https://education.github.com/pack/offers). Here you can claim the other benefits if you want.

**Comment by samus-aran** [comment]
Hey @carloss-vk  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
